### PR TITLE
Add Sample

### PR DESCRIPTION
**Pull request by dblenkus**

### CODE REVIEW

File: resolwe_bio/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+default_app_config = 'apps.BaseConfig'  # pylint: disable=invalid-name
```
Comment (dblenkus) [inline resolwe_bio/__init__.py]: @kostko: Is this the right path or should it be `resolwe-bio.apps.BaseConfig`?

Comment (kostko) [inline resolwe_bio/__init__.py]: No, it should be `resolwe_bio.apps.BaseConfig`.

**Comment by tjanez** [comment]
Please, rebase and add a CHANGELOG entry

File: resolwe_bio/signals.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+from guardian import shortcuts
+
+from resolwe.flow.models import Data, DescriptorSchema, iterate_schema
+from .models import Sample
+
+
+@receiver(post_save, sender=Data)
+def add_post_save_handler(sender, instance, **kwargs):
+    if kwargs['created'] and instance.process.flow_collection:
```
Comment (mstajdohar) [inline resolwe_bio/signals.py]: In the future we should replace this parameter with a more general group of parameters for Resolwe extensions. For example, `instance.process.ext.bio.sample`.

**Comment by mstajdohar** [comment]
LGTM, @dblenkus please accept.

**Pull request merged**